specs/models/EntityModelDescriptor.v1.json

```diff
@@ -0,0 +1,109 @@
+{
+  "title": "EntityModelDescriptor",
+  "type": "object",
+  "description": "JSON schema representation of a MarkLogic Entity Services model",
+  "x-marklogic": {
```

Comment:
This is great! Being able to extend the APIs with ML specific opens a lot of possibilities for automated testing driven by these API documents.